client/components/navbar.js

```diff
@@ -2,22 +2,50 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { logout } from '../store';
+import history from '../history';
+import { fetchUserFunction, addUserFunction } from '../crud/user';
+const { db } = require('../../firebase/initFirebase');
 
 const { auth } = require('../../firebase/initFirebase');
```

Comment:
You can include `db` and `auth` in one line. I think we should keep consistency between using `import` and `require` but there are some issues regarding that. I'll bring it up with Omri during code review.